lib/active_merchant/billing/gateways/merchant_e_solutions.rb

```diff
@@ -28,7 +28,9 @@ def initialize(options = {})
       def authorize(money, creditcard_or_card_id, options = {})
         post = {}
         post[:client_reference_number] = options[:customer] if options.has_key?(:customer)
-        post[:moto_ecommerce_ind] = options[:moto_ecommerce_ind] if options[:moto_ecommerce_ind]
+        post[:moto_ecommerce_ind] = options[:moto_ecommerce_ind] if options.has_key?(:moto_ecommerce_ind)
+        post[:recurring_pmt_num] = options[:recurring_pmt_num] if options.has_key?(:recurring_pmt_num)
+        post[:recurring_pmt_count] = options[:recurring_pmt_count] if options.has_key?(:recurring_pmt_count)
```

Comment:
Small nit, but I think we should just check if the value exists implicitly just with `if options[:moto_ecommerce_ind]`, only because this is the predominant way we check if a hash has a value.

Reply:
Done